modules/aggregation-layers/src/utils/scale-utils.js

```diff
@@ -110,7 +110,12 @@ export function getQuantileScale(domain, range) {
   while (++i < n) {
     thresholds[i - 1] = threshold(sortedDomain, i / n);
   }
-  return value => quantileScale(thresholds, range, value);
+  function scale(value) {
```

Comment:
Can you seperate out this utility method change into a different PR and add unit tests.